pages/reference.js

```diff
@@ -33,6 +36,11 @@ export default function WordsPage() {
               ))}
           </Box>
         </TableCell>
+        <TableCell>
+          <Box className={braille_styles.result_input_char}>
```

Comment:
Přijde mi názornější, když jsou vidět nejen černé tečky, ale i obdélník vytyčující plátno braillova písma, jako je to na stránce s Braillem.